package.json

```diff
@@ -7,28 +7,30 @@
   "author": "Simon Højberg <r.hackr@gmail.com>",
   "license": "MIT",
   "scripts": {
-    "build-example": "tsc; browserify examples/LoginForm/Main.js -o examples/LoginForm/public/bundle.js -t [ babelify --presets [ @babel/preset-env @babel/preset-react ] ]",
-    "test": "jest",
+    "build-example": "tsc; browserify examples/LoginForm/Main.js -o examples/LoginForm/bundle.js -t [ babelify --presets [ @babel/preset-env @babel/preset-react ] ]",
+    "test": "rm -rf dist; rm -rf build; jest",
```

Comment:
I need to get this build folder etc out of the main package and into the examples folder at some point.